packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -500,22 +519,25 @@ export const startVaultFactory = async (
     }),
   );
 
-  const { creatorFacet: governorCreatorFacet, instance: governorInstance } =
-    await E(zoe).startInstance(
-      contractGovernorInstallation,
-      undefined,
-      governorTerms,
-      harden({
-        electorateCreatorFacet,
-        governed: {
-          feeMintAccess,
-          initialPoserInvitation,
-          initialShortfallInvitation,
-          marshaller,
-          storageNode,
-        },
-      }),
-    );
+  const {
+    creatorFacet: governorCreatorFacet,
+    instance: governorInstance,
+    adminFacet,
```

Comment:
Hwo will this style work with multiple PSMs?

Reply:
It won't. hence #6146